src/styles/blocks/card

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: var(--width-card);
+  height: var(--height-card);
+  border-radius: 5px;
+  border: 1px solid var(--color-white2);
+  box-sizing: border-box;
+  background: var(--color-white);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.card__title {
+  width: 166px;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  color: var(--color-dark-blue);
+  text-align: left;
+  margin: 0 auto 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  color: var(--color-gray);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  padding: 0 84px 16px 16px;
+}
+
+.card__mark,
+.card__price {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  width: 166px;
+}
+
+.card__mark {
+  margin: 0 auto 24px;
+}
+
+.card__mark-stars {
+  align-items: center;
+}
+
+.card__mark-reviews {
+  text-align: right;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--color-dark-blue);
+}
+
+.card__price {
+  width: 166px;
+  height: 18px;
+  margin: 0 auto 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+  text-align: right;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  color: var(--color-dark-blue);
+}
+
+.card__buy {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: var(--color-blue);
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--color-white);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 16px;
+}
+
+.card__buy:hover {
+  background-color: var(--color-white);
+  color: var(--color-blue);
+  border: 1px solid var(--color-blue);
+  width: 164px;
+  height: 38px;
+}
```

Comment:
all of those can be nested inside .card selector - please use `&` [operator ](https://css-tricks.com/the-sass-ampersand/)to access them